src/validation-schemas/meta-schema.ts

```
@@ -13,6 +13,18 @@ export const xFormulaSchema: Schema = {
   additionalProperties: false,
 };
 
+// When x-formula is present, readOnly must be true
+export const xFormulaRequiresReadOnly: Schema = {
+  if: {
+    properties: { 'x-formula': { type: 'object' } },
+    required: ['x-formula'],
+  },
+  then: {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)